tools/cli/util.go

```diff
@@ -729,6 +729,15 @@ func newContextForLongPoll(c *cli.Context) (context.Context, context.CancelFunc)
 	return newContextWithTimeout(c, defaultContextTimeoutForLongPoll)
```

Comment:
Is this getting used in other places?  Can we switch all the places to use the new API you added?

Reply:
this can be a good thing to do. In the current PR haven't changed the other places so that the PR only does one thing and doesn't accidentally break what works.

Created a 'chore' story to do that change in a separate PR and test